bin/nexus-blobstores-create

```diff
@@ -49,12 +49,12 @@ while [[ $# -gt 0 ]]; do
             [[ -z $S3_SECRET_KEY ]] || yq w -i "$tmpfile" 'bucketConfiguration.bucketSecurity.secretAccessKey' "$S3_SECRET_KEY"
             [[ -z $S3_ENDPOINT ]]   || yq w -i "$tmpfile" 'bucketConfiguration.advancedBucketConnection.endpoint' "$S3_ENDPOINT"
         else
-            echo >&2 "ERROR Unknown blobstore type: ${blobstore_name}. Exiting"
+            echo >&2 "error: unknown blobstore type: ${blobstore_name}"
             exit 1
         fi
 
         # First try to create
-        echo -n >&2 "DEBUG Creating ${blobstore_type} blobstore: ${blobstore_name} ... "
+        echo -n >&2 "Creating ${blobstore_type} blobstore: ${blobstore_name}... "
```

Comment:
IUF will not be able to recognize this message if DEBUG is removed. End space removal is fine.